packages/slice-machine/src/features/onboarding/SharedOnboardingGuide.tsx

```diff
@@ -0,0 +1,26 @@
+import { OnboardingGuide } from "@prismicio/editor-fields";
+import { useMediaQuery } from "@prismicio/editor-ui";
+
+import { telemetry } from "@/apiClient";
+
+import { useOnboarding } from "./useOnboarding";
+
+export function SharedOnboardingGuide() {
+  const isOnboardingVisible = useMediaQuery({ min: "medium" });
+
+  const { onboarding, toggleStep, toggleGuide } = useOnboarding();
+
+  if (!onboarding || !isOnboardingVisible) return null;
```

Comment:
💡 Typically something we can propose to product is to have an event when the onboarding was not display due to an error. This could allow us to track how much onboarding guide are not visible because of network error, problem in our code etc.

Reply:
good idea

Reply:
Shouldn't we be using sentry or something similar for that?

Reply:
We could, I was thinking of Amplitude if product wants to track that as part of their data. Côme is not here so let's skip that and see later on :)